scripts/release/shared-commands/download-build-artifacts.js

```diff
@@ -3,65 +3,56 @@
 'use strict';
 
 const {exec} = require('child-process-promise');
-const {existsSync, readdirSync} = require('fs');
-const {readJsonSync} = require('fs-extra');
+const {existsSync} = require('fs');
 const {join} = require('path');
 const {getArtifactsList, logPromise} = require('../utils');
 const theme = require('../theme');
 
-const run = async ({build, cwd}) => {
+const run = async ({build, cwd, releaseChannel}) => {
   const artifacts = await getArtifactsList(build);
-  const nodeModulesArtifact = artifacts.find(entry =>
-    entry.path.endsWith('node_modules.tgz')
+  const buildArtifacts = artifacts.find(entry =>
+    entry.path.endsWith('build2.tgz')
   );
 
-  if (!nodeModulesArtifact) {
+  if (!buildArtifacts) {
     console.log(
       theme`{error The specified build (${build}) does not contain any build artifacts.}`
     );
     process.exit(1);
   }
 
-  const nodeModulesURL = nodeModulesArtifact.url;
+  // Download and extract artifact
+  await exec(`rm -rf ./build2`, {cwd});
+  await exec(
+    `curl -L $(fwdproxy-config curl) ${buildArtifacts.url} | tar -xvz`,
```

Comment:
Hey @acdlite I'm trying to execute this script on a Windows machine and it looks like it assumes Unix heavily. I'm trying to rewrite this in a more xplat fashion, but I can't figure out what `fwdproxy-config curl` is or returns. If you could point me to the package that enables this command or something, that'd be great. Thanks!

Reply:
@acdlite friendly ping